common/display/displayqueue.cpp

```diff
@@ -111,6 +111,9 @@ bool DisplayQueue::Initialize(uint32_t width, uint32_t height, uint32_t pipe,
   drmModePropertyPtr broadcastrgb_props =
       drmModeGetProperty(gpu_fd_, broadcastrgb_id_);
 
+  if (!broadcastrgb_props)
+	  goto out;
```

Comment:
To make it less confusing you can just wrap rest of the code under if (broadcastrgb_props) (except for final return value)